packages/core/components/FileDetails/index.tsx

```diff
@@ -76,79 +67,34 @@ function resizeHandleDoubleClick() {
-                        className={styles.resizeHandle}
-                        onMouseDown={(e) => resizeHandleOnMouseDown(e)}
-                        onDoubleClick={resizeHandleDoubleClick}
-                    />
-                )}
-                <div className={styles.fileDetailsContent}>
-                    <div className={styles.windowButtonsContainer}>
-                        <div className={styles.windowButtons}>
-                            {windowState.possibleActions.map((action) => (
-                                <WindowActionButton
-                                    key={action}
-                                    action={action}
-                                    onClick={() => windowDispatch({ type: action })}
-                                    width={WINDOW_ACTION_BUTTON_WIDTH}
-                                />
-                            ))}
-                        </div>
-                    </div>
-                    <Pagination
-                        className={classNames(styles.pagination, {
-                            [styles.hidden]: windowState.state === WindowState.MINIMIZED,
-                        })}
-                    />
+            <div className={styles.fileDetailsContent}>
+                <div
+                    className={styles.resizeHandle}
+                    onMouseDown={(e) => resizeHandleOnMouseDown(e)}
+                    // TODO:???
+                    onDoubleClick={resizeHandleDoubleClick}
+                >
+                    <Icon iconName="MoreVertical" />
+                </div>
+                <div className={styles.paginationAndContent}>
+                    <Pagination className={styles.pagination} />
                     <div className={styles.contentContainer}>
-                        <div
-                            className={classNames(styles.overflowContainer, {
-                                [styles.hidden]: windowState.state === WindowState.MINIMIZED,
-                            })}
-                        >
+                        <div className={styles.overflowContainer}>
                             {fileDetails && (
                                 <>
                                     <div className={styles.thumbnailContainer}>
                                         <FileThumbnail
                                             className={styles.thumbnail}
-                                            width={thumbnailWidth}
-                                            height={thumbnailHeight}
-                                            uri={thumbnailPath}
+                                            width="100%"
+                                            // height={thumbnailHeight}
+                                            uri={fileDetails?.getPathToThumbnail()}
                                         />
                                     </div>
                                     <div className={styles.fileActions}>
```

Comment:
Since we can't fully minimize this panel anymore, we might want to add some responsiveness to these action buttons? E.g., stacking
<img width="243" alt="image" src="https://github.com/AllenInstitute/aics-fms-file-explorer-app/assets/24441706/ae30d26b-819e-46fa-b150-b4efd4fc00e7">

Reply:
This is part of the responsive UI design lyndsay mocked up, will include in part 2 or in a totally separate responsive design refactor